medical_clinical_observation/models/medical_observation_code.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+FIELD_TYPE_SELECTION = [
+    ("integer", "Integer"),
+    ("float", "Float"),
+    ("char", "Text"),
+    ("selection", "Selection"),
+]
+
+
+class MedicalObservationCode(models.Model):
+
+    _name = "medical.observation.code"
```

Comment:
seria measure, no? `medical.measure`

Reply:
En HL7 lo miramos y le llamaban observation code
`Describes what was observed. Sometimes this is called the observation "name".`
https://www.hl7.org/fhir/observation-definitions.html#Observation.code

Reply:
Es verdad